src/models/User.js

```diff
@@ -30,14 +30,7 @@ const UserSchema = new mongoose.Schema(
   { timestamps: true }
 );
 
-UserSchema.plugin(uniqueValidator, { message: 'is already taken.' });
-
-UserSchema.methods.validPassword = (password) => {
-  const hash = crypto
-    .pbkdf2Sync(password, this.salt, 10000, 512, 'sha512')
-    .toString('hex');
-  return this.hash === hash;
-};
+UserSchema.plugin(uniqueValidator, { message: 'is already taken.' }).toString('hex');
```

Comment:
hey @okabamac , what the need for appending the `.toString('hex');` here ?

Reply:
I think it came in when I was resolving the conflict and I didn't really find out what it does. I'll do now and fix it. Thanks